savoirfairelinux/sous-chef
# Fix regression in order edit page.

## Fixes #440.

### Changes proposed in this pull request:

* Fix regression in the update order page and form.

### Status


### How to verify this change

* Edit an order
* Change information
* Save
* Verify that the order has been updated with you changes

### Deployment notes and migration


### New translatable strings


### Additional notes

*If applicable, explain the rationale behind your change.*

Ref: issue #440
